spec/requests/api/v1/category_products_spec.rb

```diff
@@ -12,6 +14,7 @@
       tags 'Products'
       consumes 'application/json'
       produces 'application/json'
+      security [Bearer: []]
```

Comment:
It's an open method for all users and crawled bots. We don't need authorization here

Reply:
check it